libs/common-const/src/tokens.ts

```diff
@@ -242,14 +284,16 @@ export const EURE_GNOSIS_CHAIN = new TokenWithLogo(
  * Addresses related to COW vesting for Locked GNO
  * These are used in src/custom/pages/Account/LockedGnoVesting hooks and index files
  */
-export const MERKLE_DROP_CONTRACT_ADDRESSES: Record<number, string> = {
+export const MERKLE_DROP_CONTRACT_ADDRESSES: Record<SupportedChainId, string> = {
   [SupportedChainId.MAINNET]: '0x64646f112FfD6F1B7533359CFaAF7998F23C8c40',
   [SupportedChainId.GOERLI]: '0xD47569F96AEF2ce1CE3B3805fAA0B90045faff8A',
   [SupportedChainId.GNOSIS_CHAIN]: '0x48D8566887F8c7d99757CE29c2cD39962bfd9547',
+  [SupportedChainId.SEPOLIA]: '', // TODO SEPOLIA: check it
 }
 
-export const TOKEN_DISTRO_CONTRACT_ADDRESSES: Record<number, string> = {
+export const TOKEN_DISTRO_CONTRACT_ADDRESSES: Record<SupportedChainId, string> = {
   [SupportedChainId.MAINNET]: '0x68FFAaC7A431f276fe73604C127Bd78E49070c92',
   [SupportedChainId.GOERLI]: '0x2f453f48a374Dd286d0Dc9aa110309c1623b29Fd',
   [SupportedChainId.GNOSIS_CHAIN]: '0x3d610e917130f9D036e85A030596807f57e11093',
+  [SupportedChainId.SEPOLIA]: '', // TODO SEPOLIA: check it
```

Comment:
This is probably available here https://cowservices.slack.com/archives/C064DAVKA83/p1702987616861039

Reply:
I haven't found. Asked solidy guys to deploy it